crates/orchestrator/src/database/mongodb/mod.rs

```diff
@@ -40,6 +40,13 @@ impl MongoDb {
         MongoDb { client }
     }
 
+    /// Mongodb client uses Arc internally, reducing the cost of clone.
+    /// Directly using clone is not recommended for libraries not using Arc internally.
+    /// Dev might want to pass an Arc manually.
```

Comment:
Not sure what "Dev might want to pass an Arc manually." means.

Reply:
will remove this comment : "Dev might want to pass an Arc manually.". It may create confusion for someone.